demo/docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: '3'
+
+services:
+    texture-dev:
```

Comment:
there is no `texture` service using `Dockerfile` so shouldn't that be used as well?

Reply:
Yes, it could be added. But it might be best solved by #31 (i.e. merge root and demo docker-compose files)